fury/data/tests/test_fetcher.py

```diff
@@ -0,0 +1,50 @@
+from fury.data import *
+import numpy as np
+import numpy.testing as npt
+import os
+from os.path import join as pjoin
+from aiohttp import InvalidURL
+import json
```

Comment:
pep8: import ordering.  check [here](https://peps.python.org/pep-0008/#imports) for more info